### PR TITLE
Adds !tonyfor command

### PR DESCRIPTION
**Pull request by kaylakantola**
- adds `!tonyfor` command
- updates `help` to include `!tonyfor` info
- updates readme
- installs `dotenv` package to handle local environment variables file
- updates `server.js` to read env vars from a `.env` file and use said vars to determine whether you should use the local token or the aws token

### CODE REVIEW

File: lib/command-map.js [hunk 1]
```diff
@@ -263,18 +263,18 @@ const commandMap = {
...
-              }, args[0] * 1000
-              )} else {
-              setTimeout(() => {
-                  return msg.channel.createMessage(`**${i}.............**`)
-                }, ((args[0] * 1000) - (i * 1000))
-              )
-            }
-          }, 1000
-        )}
+              if(i === 0) {
```
Comment (emmahyde) [inline lib/command-map.js:266]: I do not support 4 space indentation

File: lib/command-map.js [hunk 2]
```diff
@@ -331,4 +342,4 @@ const commandMap = {
   }
 }
 
-module.exports = { commandMap }
+module.exports = { commandMap }
```
Comment (emmahyde) [inline lib/command-map.js:345]: Please add newline

File: server.js [hunk 3]
```diff
@@ -1,10 +1,13 @@
 const Eris = require('eris')
-
+require('dotenv').config()
```
Comment (emmahyde) [inline server.js:2]: 💯

File: server.js [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
 const { getParameterStore } = require('./lib/aws-client.js')
 const { messageCreate } = require('./lib/message-create.js')
 
+const { LOCAL, DISCORD_TOKEN } = process.env
 async function resolveClient() {
-  const token = await getParameterStore('lucretia-bott-token')
+  const token = LOCAL
```
Comment (emmahyde) [inline server.js:8]: 1 line ternary please :)

File: README.md [hunk 5]
```diff
@@ -35,15 +37,33 @@ https://discord.com/developers/applications
...
 1. Okay! Now you have an Offline Test Bot in your Test Server. To get it online, you need to navigate to the Bot panel in Discord Developer Portal.
 
+## Authentication, option 1
+
 1. Under Build-A-Bot, click to reveal the token, the copy the token. Go to server.js under the root directory of your cloned repo. Change the following line, as of writing this it is on line 8:
-	```js
-	const token = 'your-token' // await getParameterStore('lucretia-bott-token')
-	``` 
+    ```js
+      const token = 'your-token' // LOCAL ? DISCORD_TOKEN : await getParameterStore('lucretia-bott-token')
```
Comment (emmahyde) [inline README.md:47]: shouldn't this instruction be changed if it can now be included in dotenv?

**Review by emmahyde — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (kaylakantola) [reply README.md:47]: yes i put it in below as option 2, i just didnt want to delete your stuff!

File: server.js (hunk 4 above)
Comment (kaylakantola) [reply server.js:8]: fine

File: lib/command-map.js (hunk 1 above)
Comment (kaylakantola) [reply lib/command-map.js:266]: LOOOL

File: lib/command-map.js (hunk 2 above)
Comment (kaylakantola) [reply lib/command-map.js:344]: for some reason (bein dumb???) i do not know what exactly you mean

File: README.md (hunk 5 above)
Comment (kaylakantola) [reply README.md:47]: LMK if you want me to and i WILL!

Comment (emmahyde) [reply README.md:47]: No one should do it this way. LOL

File: lib/command-map.js (hunk 2 above)
Comment (emmahyde) [reply lib/command-map.js:344]: oh it shows up in expanded view: 
Screen Shot 2020-09-22 at 12 39 39 PM
Put a newline after the last line. It is a GitHub standard

**Review by emmahyde — APPROVED** [review]

**Pull request merged**